packages/render-fidelity-tools/src/components/renderers/gltfpt-viewer.ts

```diff
@@ -77,41 +76,35 @@ export class PathtracingViewer extends LitElement {
   private async[$updateScenario](scenario: ScenarioConfig) {
     this[$canvas] = this.shadowRoot!.querySelector('canvas');
 
-    const dpr = resolveDpr();
-
-    const settings = {
-      pathtracing: true,
-      debugMode: 'None',
-      maxBounceDepth: 1,
-      useIBL: true,
-      autoScaleOnImport: false,
-      pixelRatio: 1.0 / dpr,  // TODO: clarify
-      autoRotate: false,
-      disableDirectShadows: true
-    };
-
-    this[$renderer] =
-        new PathtracingRenderer({'canvas': this[$canvas]!}, settings);
+    const enableControls = false;
+    this[$renderer] = new PathtracingRenderer(this[$canvas]!, enableControls);
 
     const renderer = this[$renderer];
+    renderer.autoScaleOnImport(false);
+    renderer.setPixelRatio(0.5);
```

Comment:
Why not set this the same as the other renderers?

Reply:
I needed to reduce pixel ratio because because my notebook couldn't handle higher resolutions for some of the scene. gpu memory limitation I guess. This should be set to 1.0 otherwise.